space-framework/docs/space-framework.md

```diff
@@ -0,0 +1,92 @@
+# Space Framework
+
+## terminoloogy 
+**Physical Space**, **pSpace** A thing that behaves like a Kubernetes kube-apiserver (including the persistent storage behind it) and the subset of controllers in the kube-controller-manager that are concerned with API machinery generalities (not management of containerized workloads). A kcp logical cluster is an example. A regular Kubernetes cluster is another example.
+
```

Comment:
Need to rephrase the above... hard to understand currently and better not to use thing... 
Maybe something like: 
Physical Space, pSpace - refers to an entity similar to a Kubernetes kube-apiserver, encompassing its persistent storage, along with the subset of controllers from the kube-controller-manager that focus on API machinery principles rather than the management of containerized workloads. Examples are: a kcp logical cluster and a standard Kubernetes cluster.